filelist-order.cxx:276:37: error: unable to find string literal operator ‘operator""fmt’ with ‘const char [3]’, ‘long unsigned int’ arguments

https://bugs.gentoo.org/594092

--- a/src/filelist-order.cxx
+++ b/src/filelist-order.cxx
@@ -273,7 +273,7 @@ void printItem(PAIR_COMPLETE_TYPE p,vector <OrderField*> *ofa) {
 					continue;
 				}
 				OrderField of = *ofp;
-#define case_entry(fmt,func) printf("%s"fmt,(first ? "" : " "),func_##func(mk)); break;
+#define case_entry(fmt,func) printf("%s" fmt,(first ? "" : " "),func_##func(mk)); break;
 				switch(of.type) {
 					case ST_DEV: case_entry("%lld",ST_DEV);
 					case ST_INO: case_entry("%lld",ST_INO);
